redisvl/schema/fields.py

```diff
@@ -26,6 +26,8 @@ class VectorDistanceMetric(str, Enum):
 
 
 class VectorDataType(str, Enum):
```

Comment:
Is there a way to combine this with the usage of the `VectorDataTypes` dictionary in utils?

Reply:
I removed `VectorDataTypes` dict entirely now. Just needed to import this into `utils` and do some string casting.